src/js/xhr.js

```diff
@@ -28,8 +32,8 @@
  * @param  {Function}       callback  The callback function
  * @returns {Object}                  The request
  */
-vjs.xhr = function(options, callback){
-  var XHR, request, urlInfo, winLoc, fileUrl, crossOrigin, abortTimeout, successHandler, errorHandler;
+var xhr = function(options, callback){
```

Comment:
Sounds good to me if you want to open an issue for either. I'm assuming the XHR from npm wouldn't change the API then, yeah?